addons/mail/static/tests/gif_picker/gif_picker_test.js

```diff
@@ -0,0 +1,125 @@
+/* @odoo-module */
+
+import {
+    click,
+    insertText,
+    start,
+    startServer,
+    waitUntil,
+} from "@mail/../tests/helpers/test_utils";
+import { patchUiSize, SIZES } from "../helpers/patch_ui_size";
+
+QUnit.module("gif");
+
+QUnit.test("composer should display an emoji button", async (assert) => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "" });
+    const { openDiscuss } = await start();
+    await openDiscuss(channelId);
+    assert.containsOnce($, "button[aria-label='Gifs']");
+});
```

Comment:
Name of test is incorrect with assertion: it tests presence of Gif button, not emoji button